src/services/codefixes/importFixes.ts

```diff
@@ -731,4 +732,35 @@ namespace ts.codefix {
             }
         }
     }
+
+    export function moduleSymbolToValidIdentifier(moduleSymbol: Symbol): string {
+        return moduleSpecifierToValidIdentifier(removeFileExtension(getBaseFileName(moduleSymbol.name)));
+    }
+
+    function moduleSpecifierToValidIdentifier(moduleSpecifier: string): string {

```

Comment:
pass in the compilerOptions, and use the target when calling `isIdentifierStart`